lib/workers/repository/update/branch/index.ts

```diff
@@ -625,7 +625,7 @@ export async function processBranch(
     }
     if (commitSha) {
       const action = branchExists ? 'updated' : 'created';
-      logger.info({ commitSha }, `Branch ${action}`);
+      logger.info({ commitSha, action }, 'Branch processing complete');
```

Comment:
Not sure about this one.

I think we can keep it as is, cause users would either want to search for created or updated branches rather than for the processing complete string.

Reply:
Yes, revert and keep as-is